homeassistant/components/netgear/device_tracker.py

```diff
@@ -111,12 +111,15 @@ def scan_devices(self):
                 )
             )
             if tracked:
-                devices.append(dev.mac)
-                if (
-                    self.tracked_accesspoints
-                    and dev.conn_ap_mac in self.tracked_accesspoints
-                ):
-                    devices.append(dev.mac + "_" + dev.conn_ap_mac)
+                # if link_rate is None, the device is still remembered by the router however,
+                # it is no longer connected. Therefore, we will not report it as seen.
+                if dev.link_rate is not None:
```

Comment:
This check can be combined with the check above, without changing the value of `tracked`. Just combine them with `and`.